collector/generatorreceiver/topos/hipster_shop.yaml

```diff
@@ -443,3 +450,9 @@ rootRoutes:
   - service: frontend
     route: /checkout
     tracesPerHour: 480
+
+incidents:
```

Comment:
what is the difference between this incident and flags?

Reply:
Good question! I don't think there's any _essential_ difference. One of the approaches I considered for this PR was eliminating incidents and letting flags have a "parent" instead, but there were a few minor complications that swayed me towards the current approach:
- conceptually it may be easier to have incidents be separate, and to think of incident design as a sequence of flag changes rather than a tree
- because of how we developed incidents initially, there was already a facility for them to be implicit, created by reference from a flag. I'm not sure this is actually a good thing, though :)
- what do we do if someone creates a recursive flag reference? Maybe it's fine but it might be very confusing.

I'm not sold on the current approach - all of these concerns can certainly be addressed, and I think it would be cleaner code-wise to go with flags only. Ultimately this way is just the one that I got better traction with. I think I'll take a little time and see how hard it would be to convert the current PR, though.